### ISSUE TITLE
Development version of CMG portal?

### ISSUE THREAD

**Issue by rsignell-usgs** [USGS-CMG/usgs-cmg-portal#271]
I see dev version of the SECOORA portal here:
http://dev.axiomdatascience.com/?portal_id=47

I see our USGS sister portal (photo and video portal) here:
http://dev.axiomdatascience.com/?portal_id=42

Is there a dev version of the CMG data portal we can explore?

**Comment by kwilcox** [comment]
portal_id=35 will get you there

**Comment by rsignell-usgs** [comment]
@kwilcox , yep, this works:
http://dev.axiomdatascience.com/?portal_id=35
But apparently not really ready for testing, right?
I was going to try the new "data view" feature for our historical time series data, but that doesn't appear possible yet because I can't select the sensor type I want, even if I add the time series data to the map:
http://dev.axiomdatascience.com/?portal_id=35&ls=383d0fca-c04b-f1e6-edb3-868fd7c61d4a#map

Just spins and doesn't bring up the rest of the selection menus (at least on Chrome):
![2018-03-05_8-32-09](https://user-images.githubusercontent.com/1872600/36977827-81d605e8-2050-11e8-82d7-c6ed0a38ac6c.png)

**Comment by kwilcox** [comment]
That is correct - right now this development version only has access to real time data (stations reporting in the last 30 days). This of course will be changed before release. I can let you know when historic data is available.
